tutorial/03-responding-to-slack-events.md

```diff
@@ -11,8 +11,8 @@ The code for this step is available [here](PythOnBoardingBot).
 
 ```
 slack_sdk>=3.0
-slackeventsapi>=2.1.0
-Flask>=1.1.2
```

Comment:
For review: Removed both slackeventsapi and flask dependencies in favor of slack_bolt only.